### PR TITLE
Changes in Readme.md

### PR DESCRIPTION
**Pull request by shaggyyy2002**
Added new resources for Computer Networking and Git-Github. (0-100) complete for free.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,8 +64,6 @@ This collection provides a list of educational resources for Software Engineers.
 # Computer Programming
 <img src="https://image.flaticon.com/icons/png/128/2620/2620900.png" height="60px"/>
 
-> If you think there is a resource which you think others would benefit on, fork, and your changes, and make a pull request!
```
Comment (CodingSpecies) [inline README.md:67]: Please can you add line 67 and 68 back please @shaggyyy2002 .

File: README.md [hunk 2]
```diff
@@ -128,6 +127,7 @@ complete beginners.
 ### 2. Video Courses - Here are 2 from our range :
```
Comment (CodingSpecies) [inline README.md:127]: Please change `2. Video Courses - Here are 2 from our range :` to `2. Video Courses - Here are 3 from our range :` @shaggyyy2002 . Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Review by CodingSpecies — APPROVED** [review]
Thank you for adding the changes @shaggyyy2002

**Review by AvidCoder101 — APPROVED** [review]

**Comment by CodingSpecies** [comment]
@all-contributors please add @shaggyyy2002  for content

**Pull request merged**